Assets/BossRoom/Scripts/Game/Client/State/ClientMainMenuState.cs

```diff
@@ -59,7 +59,6 @@ async void InjectDependenciesAndInitialize(AuthenticationServiceFacade authServi
 
             if (string.IsNullOrEmpty(Application.cloudProjectId))
             {
-                PopupManager.ShowPopupPanel("Unity Gaming Services ProjectID not set up", "Click the Readme file in the Assets Folder within the Project window in-editor to follow \"How to set up Unity Gaming Services\"");
```

Comment:
Are there any plans to put this back somewhere? If users are doing the lobby flow, I'd assume they'd need to get that popup there if they haven't setup their project, no?

Reply:
Good point! I replaced it with a tooltip instead, that shows up when hovering the disabled "Start with Lobby" button